chore(docs): replace Fx with fx and apply bold formatting.

Suraj-kumar00 opened the pull request:
This PR resolve the issue #10932


This is a draft PR that i'm oppening to check that am i going right?

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/ToolJet/ToolJet?pullRequest=10947) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/ToolJet/ToolJet?pullRequest=10947) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/ToolJet/ToolJet?pullRequest=10947) it.</sub>

Suraj-kumar00 commented:
Updated the pagination **component**, **formatting** and **content**:

- Removed 24px padding-bottom from h2 headers and adjusted div padding
- Replaced 'widget' with 'component' across pagination documentation
- Updated 'On Page Change' event description for clarity and consistency
- Added divider before 'Styles' section for better structure"

PriteshKiri commented:
Hey @Suraj-kumar00,

It looks like there are changes in 96 files, but the issue mentions updates should be made in just two files (as specified in the issue).

I'd recommend creating a new PR with changes limited to those two files. Feel free to reach out if you have any questions or need further clarification.

Suraj-kumar00 commented:
> Hey @Suraj-kumar00,
> 
> It looks like there are changes in 96 files, but the issue mentions updates should be made in just two files (as specified in the issue).
> 
> I'd recommend creating a new PR with changes limited to those two files.
> 
> Whenever you are making a new PR, just make sure to update the **`fx`** to **fx** (bold without the backticks)
> 
> Feel free to reach out if you have any questions or need further clarification.

Got it, will be opening the PR soon.
